<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SL-1352" ignored="0" name="Command line test" tests="2" failures="2" timestamp="13 Aug 2019 20:22:35 GMT" time="0.184" errors="0">
  <testcase name="@AfterMethod tearDown" time="0.0" classname="com.saucelabs.example.TestSetup">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.saucelabs.example.TestSetup.tearDown(TestSetup.java:116)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="com.saucelabs.example.TestSetup">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.saucelabs.example.TestSetup.tearDown(TestSetup.java:116)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="StanFAndroidppiumDemo" time="0.038" classname="com.saucelabs.example.TestSetup">
    <failure type="java.net.MalformedURLException">
      <![CDATA[java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:634)
	at java.net.URL.<init>(URL.java:497)
	at java.net.URL.<init>(URL.java:446)
	at com.saucelabs.example.TestSetup.createDriver(TestSetup.java:69)
	at com.saucelabs.example.TestSetup.StanFAndroidppiumDemo(TestSetup.java:78)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:539)
	... 19 more
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- StanFAndroidppiumDemo -->
  <testcase name="StanFAndroidppiumDemo" time="0.038" classname="com.saucelabs.example.TestSetup">
    <failure type="java.net.MalformedURLException">
      <![CDATA[java.net.MalformedURLException
	at java.net.URL.<init>(URL.java:634)
	at java.net.URL.<init>(URL.java:497)
	at java.net.URL.<init>(URL.java:446)
	at com.saucelabs.example.TestSetup.createDriver(TestSetup.java:69)
	at com.saucelabs.example.TestSetup.StanFAndroidppiumDemo(TestSetup.java:78)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at java.net.URL.<init>(URL.java:539)
	... 19 more
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- StanFAndroidppiumDemo -->
</testsuite> <!-- Command line test -->
